src/data-sources/salesforce-b2c/SalesforceB2CSettings.tsx

```diff
@@ -86,6 +98,18 @@ export const SalesforceB2CSettings = ( {
 					__next40pxDefaultSize
 				/>
 
+				<TextControl
+					type="text"
+					label={ __( 'Site ID', 'remote-data-blocks' ) }
+					onChange={ siteId => {
+						handleOnChange( 'site_id', siteId ?? '' );
+					} }
+					value={ state.site_id ?? 'RefArchGlobal' }
```

Comment:
Provided a default value similar to what we had before of `RefArchGlobal`. Figured since there isn't exactly an API to get a list of this, I've left it as a textbox with a helpful value.